src/main/UI/StrengthCheckerBox.java

```diff
@@ -0,0 +1,51 @@
+package UI;
+
+import controller.StrengthCheckerUIControl;
+
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class StrengthCheckerBox implements ActionListener {
```

Comment:
Instance variable be private, not package-private. Are they accessed from outside this class?